flex-config/ui_attributes.common.json

```diff
@@ -52,7 +52,9 @@
       },
       "pause_recording": {
         "enabled": true,
-        "includeSilence": false
+        "include_silence": false,
+        "indicator_banner": false,
+        "indicator_permanent": true
```

Comment:
I wasn't sure what to use for defaults here. Personally, I find Flex notifications annoying, so I went with the permanent label only. Open to changes here!